src/peft/tuners/lora/dora.py

```diff
@@ -133,12 +133,13 @@ def __repr__(self) -> str:
         return "lora.dora." + rep
 
 
-class DoraConv2dLayer(DoraLinearLayer):
+class DoraConvNdLayer(DoraLinearLayer):
```

Comment:
I wonder if we should not keep `DoraConv2dLayer` around for backwards compatibility, and do the same changes as with IA³ and LoRA with a base class and 2 subclasses.